packages/discovery-provider/integration_tests/tasks/test_get_feed_es.py

```diff
@@ -83,15 +83,15 @@ def test_get_feed_es(app):
 
     # run indexer catchup
     time.sleep(1)
-    # logs = subprocess.run(
-    #     ["npm", "run", "catchup:ci"],
-    #     env=os.environ,
-    #     capture_output=True,
-    #     text=True,
-    #     cwd="es-indexer",
-    #     timeout=30,
-    # )
-    # logging.info(logs)
+    logs = subprocess.run(
```

Comment:
what the

Reply:
no idea what this is actually, should we just remove?

Reply:
Test flow is:

* populate postgres with some fixture data
* run es-indexer to index fixture data into ES
* run tests

So that subprocess thing shells out to node to run es-indexer